public_html/wp-content/plugins/camptix-attendance/addons/attendance-ui.php

```diff
@@ -88,13 +90,15 @@
 					<span>Loading...</span>
 				</li>
 			</ul>
+			<div class="qr-scanner"><div id="qr-reader"></div></div>
 		</div>
 	</script>
 
 	<script id="tmpl-attendee-search" type="text/template">
 		<a href="#" class="close dashicons dashicons-no"></a>
 		<div class="wrapper">
 			<input type="text" autocomplete="off" placeholder="Search" />
+			<div class="previews"></div>
```

Comment:
not sure why this is here. It's also referenced in the scss above. I'll keep it for now